src/storage/__tests__/migrations-test.js

```diff
@@ -128,7 +139,9 @@ describe('migrations', () => {
     // whether any properties outside `storeKeys` are present or not.
     [
       'check dropCache at 55',
-      { ...endBase, migrations: { version: 54 }, mute: [], nonsense: [1, 2, 3] },
+      // Just before the `dropCache`, plus a `cacheKeys` property, plus junk.
+      { ...base52, mute: [], nonsense: [1, 2, 3] },
```

Comment:
Hmm, I guess I should have left that `migrations` bit in for explicitness. To check the dropCache at 55, and live up to this comment, we'll want to start just before that dropCache -- so at version 54.

(Before rebasing, the dropCache was at 53 and so the `migrations` bit would have said version 52, redundant with `base52`. So I left it out, but that made things less explicit.)